scripts/src/utils/utils.ts

```diff
@@ -163,3 +159,28 @@ export function readUTCDate(date: string | Date): Date {
 export function getAppIdFromRequest(req: RequestWithContext): string {
 	return req.context && req.context.user ? req.context.user.appId : "";
 }
+
+export async function throwOnRateLimit(appId: string, type: string, limit: number, duration: Duration) {
```

Comment:
why is this one named differently?
that is, why does it start with `throwOn` instead of `check` like the other two?
(i don't care that much, just wondering)